src/components/Permissions/List.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import { styled, theme } from '@aragon/ui'
+import ListItem from './ListItem'
+import ColumnSection from '../ColumnSection/ColumnSection'
+
+const List = ({ title, items, onClick }) => (
+  <ColumnSection title={title} shiftTitle={true}>
+    <StyledList clickable={Boolean(onClick)}>
+      {items.map(({ id, label, badge }) => (
+        <ListItem
+          key={id}
+          id={id}
+          label={label}
+          badge={badge}
+          onClick={onClick}
+        />
+      ))}
+    </StyledList>
+  </ColumnSection>
+)
+
+const StyledList = styled.ul`
+  list-style: none;
+  background: ${theme.contentBackground};
+  border: 1px solid ${theme.contentBorder};
+  border-radius: 4px;
+  li {
+    display: flex;
+    align-items: center;
+    overflow: hidden;
+    padding: 19px;
+    border-top: 1px solid ${theme.contentBorder};
+    cursor: ${({ clickable }) => (clickable ? 'pointer' : 'default')};
+  }
+  li:first-child {
+    border-top: 0;
+  }
+`
+
+export default List
```

Comment:
Yes, it is for now a specific component for Permissions, but the idea is to make it a shared component (by removing the label / badge things) in Aragon UI when its name / implementation / API is good enough.

I think it would be nice to have a `List` (or approaching name) in Aragon UI, and I agree that it should be a wrapper around `Table`, except if we have a strong reason to use a `<ul>` instead (e.g. unsolvable CSS issue with a `<table>`, accessibility).